ui/src/pub.vue

```diff
@@ -465,6 +467,11 @@ export default {
             this.$router.push('/project/'+project_id);
         },
 
+        redirect(funding) {
+            if(funding.funder =='NSF') return "https://www.nsf.gov/awardsearch/showAward?AWD_ID="+funding.id.slice(funding.id.lastIndexOf('-') +1);
+            if(funding.funder =='NIH') return "https://reporter.nih.gov/project-details/"+funding.id.slice(funding.id.lastIndexOf('U'));
```

Comment:
This seems a bit brittle.. Could you add comments on what the NSF/NIH IDs look like and why this combination of slide/lastIndexOf can turn the ID into properly formatted URL? 

Also, this might warrant the use of regex.match?

Reply:
sure ! 
An NSF ID usually contains an identifier and an id number. The identifier part signifies that funding is received from which NSF institution/organization and the number part is the id which is separated by a '-'. This numeric part can be added to the link. 

For example, 
IIS-1636893 is NSF FUNDING data on a publication on brainlife. Then, we can use the numeric part to redirect to 
https://www.nsf.gov/awardsearch/showAward?AWD_ID=1636893

Similarly, in NIH links I am able to see a pattern with NIH where any string before U signifies the institution and the rest is project identifier.

For example, NIH NIMH U01MH097435 => https://reporter.nih.gov/project-details/U01MH097435

Reply:
Thanks for the info! Could you add this info to the source code so we know why it's using the logic that it's using? 

If you want to make this a bit more robust, you should probably do something like this

```
matches = funding.id.match(/IIS-([0-9]+)/)
if(match) { ... match[1] should be the parsed out id) } else { //no match! redirect somewhere else?}
```